package ru.mirea.elkinasa.catbreeds.domain.models;

public class CatBreed {
    private final String id;
    private final String name;
    private final String description;
    private final String origin;
    private final String imageUrl;

    public CatBreed(String id, String name, String description, String origin, String imageUrl) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.origin = origin;
        this.imageUrl = imageUrl;
    }

    public String getId() { return id; }
    public String getName() { return name; }
    public String getDescription() { return description; }
    public String getOrigin() { return origin; }
    public String getImageUrl() { return imageUrl; }
}
